eppo_metrics_sync/eppo_metrics_sync.py

```diff
@@ -15,6 +15,16 @@
 API_ENDPOINT = 'https://eppo.cloud/api/v1/metrics/sync'
 
 
+def _safer_load(path):
```

Comment:
In what sense is this a "safer" load? It looks like this is basically a wrapper for `yaml.safe_load` with a context handler and some specific error-handling logic.

I usually create a file called `helpers.py` to contain functions like that I'd call this `load_yaml`. I wouldn't use a leading underscore because this function could potentially be used in multiple places throughout the repo as we expand and refactor.